lib/server.js

```diff
@@ -189,19 +193,21 @@ function handleRequest (opts, req, resp) {
         return handleResponse(reqOpts, newReq, resp, result);
     })
     .catch (function(e) {
-        if (e && e.name !== 'HTTPError') {
-            var stack = e.stack;
+        if (!e || e.name !== 'HTTPError') {
```

Comment:
This was a problem I have seen long ago but haven't gotten around to fix. Yey :)